js/script.js

```diff
@@ -0,0 +1,55 @@
+const buttonAddBook = document.getElementById('button-add');
+const containerBooks = document.getElementById('content-books');
+const formBooks = document.getElementById('form-books');
+let inputTitle = '';
+let inputAuthor = '';
+let newBook; let newData; let allBooks;
+const books = {
+  show_books() {
+    containerBooks.innerHTML = '';
+    this.get_localstorage().forEach((book, index) => {
+      containerBooks.innerHTML
+            += `<div class='card-book' id='book-${index}'><strong class='book-title'>${book[0]}</strong>`
+            + `<p class='book-author'>${book[1]}</p>`
+            + `<button type='button' id='button-remove-${index}' onclick='books.remove_book_by_index(${index})'>Remove</button></div><hr>`;
+    });
+  },
+  add_new_book() {
+    inputTitle = document.getElementById('input-title').value;
+    inputAuthor = document.getElementById('input-author').value;
+    newBook = [[inputTitle, inputAuthor]];
+    // newRegister = this.get_localstorage()
```

Comment:
Kindly delete commented code, you can use git history instead if you need your old code.

I trust you can do this before merging.